nbextensions/bulkImportCell/testAppObj-ci.js

```diff
@@ -0,0 +1,487 @@
+define(['./jobsData'], (JobsData) => {
+    'use strict';
+
+    const jobData = JobsData.allJobs;
+    const jobRemapping = {
```

Comment:
These jobIDs are from the narrative https://ci.kbase.us/narrative/57559 / "Many cell types" on localhost.